linux/net/rina/ipcp-instances.h

```diff
@@ -94,9 +94,19 @@ struct efcp_config {
         /* The data transfer constants */
         struct dt_cons * dt_cons;
 
+        /* Left here for phase 2 */
         struct policy * unknown_flow;
 };
 
+/* Represents the configuration of the RMT */
+struct rmt_config {
+	/* The PS name for the PDU Forwarding Table (Next hop) */
+	struct policy * pft_policy_set;
+
+	/* The PS name for the RMT */
+	struct policy * rmt_policy_set;
+};
+
 /* Represents a DIF configuration (policies, parameters, etc) */
```

Comment:
What's the point of storing the policy names there? Remember that policy-set names are already stored in the per-component "available policy set list", as soon as they are published by plugins.